src/pages/_app.tsx

```diff
@@ -1,10 +1,15 @@
 import '@hashicorp/platform-util/nprogress/style.css'
+import createConsentManager from '@hashicorp/react-consent-manager/loader'
 import { ErrorBoundary } from '@hashicorp/platform-runtime-error-monitoring'
 import useAnchorLinkAnalytics from '@hashicorp/platform-util/anchor-link-analytics'
 import { DeviceSizeProvider } from 'contexts'
-import BaseLayout from 'layouts/base'
+import BaseLayout from 'layouts/base-new'
 import './style.css'
 
+const { ConsentManager, openConsentManager } = createConsentManager({
```

Comment:
As mentioned in [this Asana task](https://app.asana.com/0/1100423001970639/1201646681395160/f), we'll likely need to rethink this when merging with `main` and co-locating `dot-ios`, but for now, keeping it simple.